src/ast/mod.rs

```diff
@@ -733,6 +733,17 @@ pub enum Expr {
     ///
     /// See <https://docs.snowflake.com/en/sql-reference/constructs/where#joins-in-the-where-clause>.
     OuterJoin(Box<Expr>),
+
+    /// ClickHouse supported parameterized view
+    /// for example
+    /// ```sql
+    /// CREATE VIEW view AS SELECT * FROM TABLE WHERE Column1={column1:datatype1} and Column2={column2:datatype2}
+    /// ```
+    /// see doc <https://clickhouse.com/docs/en/sql-reference/statements/create/view#parameterized-view>
+    ParameterizedViewArg {
+        columns: Ident,
```

Comment:
Since there is only a single column here, I think this would make more sense to be named

```suggestion
        column: Ident,
```

Reply:
Sure